### PR TITLE
Google Cloud Functions Funqy HTTP

### LINKED ISSUE #11299: Add documentation and quickstart for Funqy HTTP for Google Cloud Functions
**Description**
We have a Google Cloud Functions HTTP binding so we should be able to use Funqy HTTP.

The only missing part are:

- Testing that it works (as it's theorical) ;)
- Adding documentation
- Adding quickstart

### PR DESCRIPTION
**Pull request by loicmathieu**
Document how to use funqy-http with Google Cloud Functions and add an example in the integration test.

Fixes #11299

### CODE REVIEW

File: docs/src/main/asciidoc/funqy-gcp-functions-http.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Funqy HTTP Binding with Google Cloud Functions
+:extension-status: experimental
+
+include::./attributes.adoc[]
+
+If you want to allow HTTP clients to invoke on your Funqy functions on Google Cloud Functions, Quarkus allows you to expose multiple
```
Comment (gsmet) [inline docs/src/main/asciidoc/funqy-gcp-functions-http.adoc:11]:
```suggestion
If you want to allow HTTP clients to invoke your Funqy functions on Google Cloud Functions, Quarkus allows you to expose multiple
```

File: docs/src/main/asciidoc/funqy-gcp-functions-http.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+The solution is located in the `funqy-google-cloud-functions-http-quickstart` {quickstarts-tree-url}/funqy-quickstarts/funqy-google-cloud-functions-http-quickstart[directory].
+
+== The Code
+
+There is nothing special about the code and more importantly nothing Google Cloud specific.  Funqy functions can be deployed to many different
+environments and Google Cloud Functions is one of them.  The Java code is actually the same exact code as the {quickstarts-tree-url}/funqy-quickstarts/funqy-http-quickstart[funqy-http-quickstart].
+
+== Getting Started
+
+The steps to get this quickstart running are exactly the same as defined in the link:amazon-lambda-http[Google Cloud Functions HTTP Guide].
```
Comment (gsmet) [inline docs/src/main/asciidoc/funqy-gcp-functions-http.adoc:42]: The link is for Amazon Lambda whereas the test is for Google Cloud.

File: docs/src/main/asciidoc/funqy-gcp-functions-http.adoc [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+This differences are that you are running from a quickstart and the maven dependencies are slightly different.
```
Comment (gsmet) [inline docs/src/main/asciidoc/funqy-gcp-functions-http.adoc:43]:
```suggestion
This differences are that you are running from a quickstart and the Maven dependencies are slightly different.
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Spotted a couple of small issues in the doc.

File: docs/src/main/asciidoc/funqy-gcp-functions-http.adoc (hunk 2 above)
Comment (loicmathieu) [reply docs/src/main/asciidoc/funqy-gcp-functions-http.adoc]: Fixed

**Review by gsmet — APPROVED** [review]

**Pull request merged**